lib/perl/Genome/Disk/Command/Volume/SyncUsage.pm

```diff
@@ -27,44 +25,30 @@ class Genome::Disk::Command::Volume::SyncUsage {
             doc => 'Sync unallocated_kb?',
         },
     },
-    has_transient => {
-        tie_stderr => {
-            is => 'Boolean',
-            default => 1,
-            doc => '(warning) globally tie STDERR to this logger',
-        },
-    },
-    doc => 'Sync usage info for volume (e.g. total KB and unallocated KB)',
+    doc => 'Sync total and unallocated KB usage for volumes',
 };
 
 sub help_detail {
-    'Sync usage info for volume (e.g. total KB and unallocated KB).'
+    return __PACKAGE__->__meta__->doc;
 }
 
 sub execute {
     my $self = shift;
+    $self->status_message('Disk volume sync usage...');
 
-    my %args = (verbose => 1);
     for my $volume ( $self->volumes ) {
-        my $transaction = UR::Context::Transaction->begin;
-        try {
-            $self->info('Syncing volume: '.$volume->mount_path);
-            if ($self->total_kb)      {
-                $self->debug('Sync total kB...');
-                $volume->sync_total_kb(%args);
-            }
-            if ($self->unallocated_kb) {
-                $self->debug('Sync unallocated kB...');
-                $volume->sync_unallocated_kb(%args);
-            }
-            $transaction->commit or die 'Failed to commit volume! '.$volume->mount_path;
+        $self->status_message('Syncing volume: %s', $volume->mount_path);
+        for my $type (qw/ total unallocated /) {
```

Comment:
Sure. The previous version had status messages in the command and volume.pm I hesitated using this function because then the types of things to sync (total./unallocated) would need to be passed in. That would create 2 places where the params live and wouldneed to be maintined. One in the command and the other the function in volume.pm.